api/runnerpool/placer_config.go

```diff
@@ -13,9 +13,9 @@ type PlacerConfig struct {
 	PlacerTimeout time.Duration `json:"placer_timeout"`
 }
 
-func NewPlacerConfig() PlacerConfig {
+func NewPlacerConfig(timeout time.Duration) PlacerConfig {
```

Comment:
See oci lb, main.go to see how this would work there.. NewPlacerConfig() provides reasonable defaults. For tests, these fields are already public and you can easily override them.

Reply:
I added a new public field so we don't change the signature for NewPlacerConfig